coursedata/adventures/nl.yaml

```diff
@@ -25,7 +25,7 @@ adventures:
                     ```
                     ask De hoofdpersoon van dit verhaal is
                     print De hoofdpersoon gaat nu in het bos lopen
-                    echo Hij is wel een beetje bang, die 
+                    echo Hij is wel een beetje bang, die
```

Comment:
Ah this made me realize ewe have a small error in echo, thanks (this one: https://github.com/Felienne/hedy/issues/1258)